src/main/madara.ts

```diff
@@ -151,6 +151,21 @@ export async function start(window: BrowserWindow, config: MadaraConfig) {
     throw Error('Node is already running!');
   }
 
+  // Check if release exists and setup
+  if (!await releaseExists(config)) {
+    await setup(window, config);
+  }
+
+  // Run the setup command for the node
+  const setupArgs = ['setup', '--chain', 'dev', '--from-local', './configs'];
```

Comment:
We just download the binary file, so `--from-local ./configs` shouldn't make sense because we never downloaded the config locally.

Reply:
`--chain dev` will initialize the config inside the dev folder. So if the user starts without development mode, this would cause the node to fail as it won't find the config. Can you verify?